src/components/NavBar/SearchBar.tsx

```diff
@@ -131,10 +155,8 @@ export const SearchBarDropdown = ({ toggleOpen, tokens, collections, hasInput }:
   })
 
   const trendingTokens = useMemo(() => {
-    // TODO reimplement this logic with NFT search
-    // return trendingTokenResults?.slice(0, isTokenPage ? 3 : 2)
-    return trendingTokenResults?.slice(0, 4)
-  }, [trendingTokenResults])
+    return trendingTokenResults?.slice(0, phase1Flag === Phase1Variant.Enabled ? (isTokenPage ? 3 : 2) : 4)
```

Comment:
can you un-nest this logic so it's a little easier to follow?

Reply:
So like how I had it before?
```
return phase1Flag === Phase1Variant.Enabled
      ? trendingTokenResults?.slice(0, isTokenPage ? 3 : 2)
      : trendingTokenResults?.slice(0, 4)
```
or are you thinking something different?

Reply:
I mean a single call to `slice`, and un-nested logic for determination of the `end` argument. 
```
let end = 4
if (phase1Flag === Phase1Variant.Enabled) {
  end = isTokenPage ? 3 : 2
}
return trendingTokenResults?.slice(0, end)
```
pref current implementation to former for the single `slice` call, but the bit about `end` determination logic structure is a nit.

Reply:
I think `let` overwrites should be used as infrequently as possible vs const when we know from init what the value should be set as. What do you think of something like this:
```
  const trendingTokensLength = phase1Flag === Phase1Variant.Enabled ? (isTokenPage ? 3 : 2) : 4

  const trendingTokens = useMemo(() => {
    return trendingTokenResults?.slice(0, trendingTokensLength)
  }, [trendingTokenResults, isTokenPage, phase1Flag])